src/assets/styles/editor.scss

```diff
@@ -367,7 +367,8 @@
 	width: 100%;
 	min-height: 40px;
 	box-sizing: border-box;
-	overflow-y: auto;
+    overflow-y: auto;
```

Comment:
seems there's two missing spaces here?

Reply:
tabs vs spaces issue -_- should be fixed!